stock_mts_mto_rule/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Florian da Costa <florian.dacosta@akretion.com>
+* Jared Kipe <jared@hibou.io>
+* Cetmix <https://cetmix.com>
```

Comment:
This should contain only individuals.

The company should appear in the 'author' manifest key. But as said above, you haven't authored this module. So, please modify this